library/src/main/java/com/github/pwittchen/reactivenetwork/library/network/observing/strategy/PreLollipopNetworkObservingStrategy.java

```diff
@@ -44,8 +45,9 @@
     final IntentFilter filter = new IntentFilter();
     filter.addAction(ConnectivityManager.CONNECTIVITY_ACTION);
 
-    return Observable.create(new Observable.OnSubscribe<Connectivity>() {
-      @Override public void call(final Subscriber<? super Connectivity> subscriber) {
+    return Observable.create(new ObservableOnSubscribe<Connectivity>() {
+      @Override public void subscribe(final ObservableEmitter<Connectivity> subscriber)
```

Comment:
Please change `subscriber` name to `emmiter`. Type was changed, so in this case, we should change variable name as well.